src/main/java/com/iota/iri/service/TipsSolidifier.java

```diff
@@ -48,6 +48,8 @@ public void init() {
 
     private void scanTipsForSolidity() throws Exception {
         int size = tipsViewModel.nonSolidSize();
+        log.info("#Solid/NonSolid: {}/{}",tipsViewModel.size()-size,size);
```

Comment:
I think that this should be a debug message.
Also  add `getNumOfSolidTips()` in `TipsViewModel`

Reply:
if left on purpose and made debug (agree w/ Gal), still this should be printed very infrequently imo.
iiuc, in the current PR this will be printed more than every second.

Reply:
shall i add  a function called .solidSize() instead? would be more consistent naming-wise with nonSolidSize() :-)

Reply:
@alon-e rgd the debug info shall i make a construct like https://github.com/iotaledger/iri/blob/a45fa1a63063aec2bd5944875eb9e0f38f0ef8a5/src/main/java/com/iota/iri/network/Node.java#L537